### Answers

1. The console will output 3 as this is the value of 'i' where the loop breaks. 'i' was declared with var so line 12 is not out of scope.
2. The console will output 150 as this is the final value of the variable, it was also declared with var so line 13 is not out of scope.
3. The console will output 150 as this is the final value of the variable.
4. The function will return [50, 100, 150], the function simply multiplies each of the prices by the discount and returns this. 
5. Error as 'i' was declared with let within a block that is out of scope at line 12.
6. Error as 'discountedPrice' was declared with let within a block that is out of scope at line 13.
7. The console will output 150 as this is the final value of the variable, it was declared with let but line 14 is within the same block as the declaration.
8. The function will return [50, 100, 150], the function simply multiplies each of the prices by the discount and returns this. 'discounted' was declared within the same block as the return.
9. Error as 'i' was declared with let within a block that is out of scope at line 11.
10.  Console will output '3' as this is the value assigned to the constant at declaration.
11.  Console will output [50, 100, 150]. The function works the same as before, the const declaration for discounted is just saying that there is a constant reference to the array but the elements can still be changed so the function still works.

12. :
A - student.name
B - student['Grad Year']
C - student.greeting()
D - student['Favorite Teacher'].name
E - student.courseLoad[0]

13. :
A - '32', integers map to their exact string representation
B - 1, the string is mapped to an integer as integers are needed fro mathematical operations
C- 
